### ISSUE TITLE
Fix up Python extension recommendations

### ISSUE THREAD

**Issue by SaschaCowley** [nvaccess/vscode-nvda#15]
Currently, our `extensions.json` lists both `ms-python.python` and `ms-python.vscode-pylance`, despite the fact that [the Python extension installs Pylance by default](https://marketplace.visualstudio.com/items?itemName=ms-python.python#installed-extensions). Since the extension works best with the [Python Debugger](https://marketplace.visualstudio.com/items?itemName=ms-python.debugpy) extension installed, we should either:
1. Remove the explicit recommendation of Pylance, and just rely on the other extensions recommended by the Python extension; or
2. Add Python Debugger to the recommendations to make our recommendation explicit.

**Comment by seanbudd** [comment]
Is the python debugger extension useful/accessible?

**Comment by michaelDCurran** [comment]
> Is the python debugger extension useful/accessible?
Absolutely. It is required to run debug sessions in VS Code for Python projects, including break points, call stacks, watching variables, runtime execution of Python code within the debugged process.

**Comment by SaschaCowley** [comment]
As Mick has said, yes. Additionally it's required (if you want to debug Python) for the remote debugging method I'm writing up. I can open a PR to include everything required for that method (`launch.json` and changes to `extensions.json`) all at once if that's preferable.

**Comment by hwf1324** [comment]
> 1. Remove the explicit recommendation of Pylance, and just rely on the other extensions recommended by the Python extension; or

With the merger of https://github.com/nvaccess/nvda/pull/17744, shouldn't we close this issue? Or recommend other pyright extensions? Maybe also clean up the settings in `settings.json` that are obscured by pyright.
